assets/src/setup/pages/template-mode/get-selection-details.js

```diff
@@ -29,12 +29,13 @@ export const NON_TECHNICAL = 'nonTechnical';
  * Returns the degree to which each mode is recommended for the current site and user.
  *
  * @param {Object} args
+ * @param {boolean} args.currentThemeIsReaderTheme Whether the currently active theme is in the reader themes list.
  * @param {boolean} args.userIsTechnical Whether the user answered yes to the technical question.
  * @param {boolean} args.hasPluginIssues Whether the site scan found plugin issues.
  * @param {boolean} args.hasThemeIssues Whether the site scan found theme issues.
  * @param {boolean} args.hasScanResults Whether there are available scan results.
  */
-export function getRecommendationLevels( { userIsTechnical, hasPluginIssues, hasThemeIssues, hasScanResults = true } ) {
+export function getRecommendationLevels( { currentThemeIsReaderTheme, userIsTechnical, hasPluginIssues, hasThemeIssues, hasScanResults = true } ) {
```

Comment:
I think `currentThemeIsReaderTheme` can be renamed to `currentThemeisAmpCompatible`, since that's what is actually being determined here.

/cc @westonruter

Reply:
It's actually both: current theme is AMP-compatible _and_ it's in the list of Reader themes.

So the name I think is OK. It could also be: `currentThemeIsAmongReaderThemes`.

Reply:
Updated to `currentThemeIsAmongReaderThemes` in 1673aa0e2 because that is more accurate.